userguide/plugins.rst

```diff
@@ -101,16 +101,20 @@ with :guilabel:`NAT`. It is recommended to keep these plugins set to
 :guilabel:`DHCP` unless manually configuring an IP address is preferred.
 
 If both :guilabel:`NAT` and :guilabel:`DHCP` are unset, an IPv4 or IPv6 address can
-be manually entered. 
+be manually entered.
 
 Click :guilabel:`ADVANCED PLUGIN INSTALLATION` to show all options for the
 plugin jail. The options are described in :ref:`Advanced Jail Creation`.
 
 To start the installation, click :guilabel:`SAVE`.
 
-Depending upon the size of the application, the installation may take a few minutes
-to download and install. A confirmation message displays once the plugin installation
-completes, along with any post-installation notes. 
+Plugin download and installation can take several minutes, depending on
+the size of the application. A confirmation message displays when the
+plugin installation completes, along with any post-installation notes.
+
+Installed plugins are immediately started. By default, all plugins are
+started when the system boots. Unsetting :guilabel:`Boot` means the
+plugin must be started manually.
```

Comment:
```suggestion
plugin will not start when the %brand% system boots, and must be started manually when needed.
```

Reply:
The changed version is somewhat redundant ("started" and "will start").  It's not clear what the difference is there.  "when the system boots" is unspecific as to which system is being talked about. The last sentence uses negative logic and an unspecific "requires": what requires what?  Is it a dependency, or something the system does?  The suggestion avoids most of these problems and is shorter.
```
A plugin is started immediately after installation and will start when the system boots. To disable boot startup and require manually starting the plugin, unset :guilabel:`Boot`.
```